src/generic/generic-component-form-legend/generic-component-form-legend.twig

```diff
@@ -1,27 +1,46 @@
+{#
+  Parameters:
+    - "content" (string) (default: '')
+    - "level" (string) (default: ''): none, "1" or "2"
```

Comment:
nitpicking:
"none, '1' or '2' " could be completed. is "none" a valid value?  
"1": Legend for a top-level fieldset 
"2": Legend for a nested fieldset